data/campaigns/Legend_of_Wesmere/scenarios/chapter3/10_Cliffs_of_Thoria.cfg

```diff
@@ -112,8 +112,8 @@
         side=5
         allow_player=no
         canrecruit=yes
-        {GOLD 260 320 380}
-        {INCOME 4 8 12}
+        {GOLD 220 300 380}
+        {INCOME 2 7 12}
```

Comment:
A little too drastic on the income change I think. You could make the gold values {200 300 400} here I guess, keeping the income as in the original.

Reply:
> A little too drastic on the income change I think. You could make the gold values {200 300 400} here I guess, keeping the income as in the original.

ok done in 241f2ca